statsd/statsd.go

```diff
@@ -470,6 +474,11 @@ func (c *Client) Flush() error {
 	return nil
 }
 
+// IsClosed returns if the client has been closed.
+func (c *Client) IsClosed() bool {
+	return c.isClosed
```

Comment:
I missed this during the first review, but we need to  lock `c.closerLock` here too.

Reply:
Done 👍